nixos/modules/config/xdg/icons.nix

```diff
@@ -13,6 +13,14 @@
         [XDG Icon Theme specification](https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html).
       '';
     };
+    xdg.icons.fallbackThemes = lib.mkOption {
+      type = lib.types.listOf lib.types.str;
+      default = [];
+      description = ''
+        Names of the fallback icon/cursor themes, in order of preference, to be used when no other icon can be found.
```

Comment:
The `default` theme is entirely cursor-spceific. In case of icons, `hicolor` is the fallback theme.

Reply:
It's a little weird because some themes also explicitly inherit `default`.

Reply:
That's indeed weird and might be a bug in them

Reply:
I see Adwaita is both cursor and icon theme. This might be the source of confusion.

Reply:
Another guess is `hicolor` might have been `default` 20 years ago.
https://gitlab.freedesktop.org/xdg/default-icon-theme/-/blob/master/spec/icon-theme-spec.xml?ref_type=heads#L940

Reply:
Looks like you are right https://web.archive.org/web/20021017204142/http://www.freedesktop.org/standards/icon-theme-spec/icon-theme-spec.html#directory_layout

Reply:
Yeah, so our options for naming here are "slightly too generic" and "slightly too specific". I think I prefer the more generic one, because it at least covers all the potential weirdness this can cause.

Reply:
> Looks like you are right https://web.archive.org/web/20021017204142/http://www.freedesktop.org/standards/icon-theme-spec/icon-theme-spec.html#directory_layout

It seems they have a leftover which might be the reason it's still alive (copypasting to newly created themes): https://gitlab.freedesktop.org/xdg/default-icon-theme/-/blob/master/spec/icon-theme-spec.xml?ref_type=heads#L676

> Yeah, so our options for naming here are "slightly too generic" and "slightly too specific". I think I prefer the more generic one, because it at least covers all the potential weirdness this can cause.

The only problem is that it might be quite confusing to the user, especially if other distros name it such as [default-cursors](https://archlinux.org/packages/extra/any/default-cursors/). At least Fedora should have a similar package and I guess it's also named with cursors in the name...

Reply:
OK so I checked a bunch of themes we ship in nixpkgs (thanks nix-index) and it looks like explicitly inheriting `default` is in fact very rare, so let's go with the cursor name.

Reply:
On Fedora, it's a part of libXcursor package
https://src.fedoraproject.org/rpms/libXcursor/blob/rawhide/f/libXcursor.spec#_69